### PR TITLE
Prepare testsys v0.0.9, tools v0.7.0 release

### PR DESCRIPTION
**Pull request by etungsten**
**Issue number:**
N/A

**Description of changes:**
Bump crate version, prep releases.

**Testing done:**
N/A

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by ecpullen — APPROVED** [review]
LGTM

**Comment by etungsten** [comment]
"Build / images" check is expected to fail since the tools image hasn't been released yet,

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,41 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+
+- Add EKS service endpoint override in the EKS resource agent ([#860])
+- Set necessary environment variables to enable new K8s version cluster creation in metal and vsphere agents ([#866])
+- Add option for EKS-A release manifest url and fetch EKS-A binary at runtime in metal and vsphere agents ([#867])
+
+### Changed
+
+- Remove `eksctl` build workaround ([#844])
+- Remove `--force-cleanup` flag from eks-a invocation in metal and vsphere agents ([#851])
+- Restrict IMDS on nodes launced by testsys-launcher ([#852])
```
Comment (stmcginnis) [inline CHANGELOG.md]: s/launced/launched/

**Review by stmcginnis — APPROVED** [review]
One minor typos, but I'll approve anyway. If you can update to fix that, that would be good. If it's easier to clean that up later, I don't think it's that critical.

**Pull request merged**